### PR TITLE
Fix bug in libpe ordinal mask handling

### PR DESCRIPTION
**Pull request by redspot**
* Fix ordinal mask bug in imports.c
* Refactor ordinal mask handling

### CODE REVIEW

File: lib/libpe/imports.c [hunk 1]
```diff
@@ -112,8 +112,8 @@ static uint32_t get_functions_count(pe_ctx_t *ctx, uint64_t offset) {
 				if (thunk_type == 0)
 					return count;
 
-				bool is_ordinal = (thunk_type & IMAGE_ORDINAL_FLAG32) != 0;
-				
+				bool is_ordinal = (thunk_type & IMAGE_ORDINAL_FLAG64) != 0;
```
Comment (redspot) [inline lib/libpe/imports.c]: This is the bug. The 64-bit case was using the 32-bit flag.

**Comment by GoGoOtaku** [comment]
Thank you for the fix

**Comment by redspot** [comment]
Is there something I need to do for the PR? The OSX build failure seems unrelated to my fix.

**Comment by GoGoOtaku** [comment]
Oh, no you're good.
I looked into the issue yesterday and didn't have the muse to look into where github puts the includes now and forgot to merge this.

**Pull request merged**